crates/core/liveview-native-core-jetpack/core/src/test/java/org/phoenixframework/liveview_jetpack/DocumentTest.kt

```diff
@@ -20,7 +20,7 @@ import org.phoenixframework.liveviewnative.core.LiveFile;
 class SocketTest {
     @Test
     fun simple_connect() = runTest {
-        var live_socket = LiveSocket("http://127.0.0.1:4001/upload?_lvn[format]=swiftui", Duration.ofDays(10));
+        var live_socket = LiveSocket.connect("http://127.0.0.1:4001/upload?_format=jetpack", Duration.ofDays(10));
```

Comment:
This is needed because [uniffi supports async constructors but the swift bindings are the only ones with primary constructor support](https://github.com/mozilla/uniffi-rs/pull/2048).